package.json

```diff
@@ -142,7 +142,7 @@
     "edge-currency-monero": "^1.0.0",
     "edge-currency-plugins": "^2.0.3",
     "edge-exchange-plugins": "^0.19.5",
-    "edge-login-ui-rn": "^2.4.0",
+    "edge-login-ui-rn": "https://github.com/EdgeApp/edge-login-ui-rn/#ab257cba0d32f8a8957e2d055842c97535270083",
```

Comment:
Makes this PR a draft

Reply:
That's fair. I knew this would conflict with the dependency once login-ui is merged which it already is so this can be rebased out.